src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  addMovie: (movie: Movie) => void;
+  listOfMovie: Movie[];
+};
+export const FindMovie: React.FC<Props> = (props) => {
+  const { addMovie, listOfMovie } = props;
+  const [findedMovie, setMovie] = useState(null as Movie | null);
+  const [query, setQuery] = useState('');
+  const [isFoundMovie, setIsFound] = useState(true);
+
+  const loadMovie = async () => {
+    const movieData = await getMovie(query);
+
+    if (movieData.Response === 'False') {
+      setMovie(null);
+      setIsFound(false);
+
+      return;
+    }
+
+    setIsFound(true);
+    setMovie({
+      title: movieData.Title,
+      description: movieData.Plot,
+      imgUrl: movieData.Poster,
+      imdbId: movieData.imdbID,
+      imdbUrl: `https://www.imdb.com/title/${movieData.imdbID}`,
+    });
+    setQuery('');
+  };
+
+  const handleSearchForm = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    setQuery(value);
+    setIsFound(true);
+  };
+
+  const handleAddMovie = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    if (findedMovie && !listOfMovie.find(item => item.imdbId === findedMovie.imdbId)) {
+      addMovie(findedMovie as Movie);
+    }
+
+    setMovie(null);
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
```

Comment:
Use classnames library here to make the red border appear only when the movie is not found